doc/Language/js-nutshell.pod6

```diff
@@ -0,0 +1,191 @@
+=begin pod :tag<convert>
+
+=TITLE Perl 6 from Node.js - Nutshell
+
+=SUBTITLE Learning Perl 6 from Node.js, in a nutshell.
+
+This page attempts to provide a way for users experienced in Node,js to learn
```

Comment:
I know you've told me twice already that you don't see a comma here, but it still reads `Node,js`.

Reply:
Whoops, thought I got that in the last commit